invoice/views.py

```diff
@@ -46,16 +123,21 @@ def form_valid(self, form):
         return super().form_valid(form)
 
 
-class CustomerUpdateView(SuccessMessageMixin, UpdateView):
+class CustomerUpdateView(OwnMixin, UpdateView):
```

Comment:
On the other views you did it like this:
```suggestion
class CustomerUpdateView(OwnMixin, SuccessMessageMixin, UpdateView):
```
Intended?